### PR TITLE
Updates instructions to install and test osgjs

### PR DESCRIPTION
**Pull request by cedricpinson**

### CODE REVIEW

**Comment by pierreant-p** [comment]
solide

**Comment by jtorresfabra** [comment]
Why not to use localhost in https://github.com/cedricpinson/osgjs/blob/master/Gruntfile.js#L379. It probably does the trick also.

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ Simply type in the command-line:
 
     grunt serve
 
-It should open a browser pointing to the root source directory, then navigate to examples or tutorials folder.
+You can now tests some real examples http://localhost:9000/examples http://localhost:9000/tutorial
```
Comment (jtorresfabra) [inline README.md]: s/tests/test/

File: website/contents/installation.md [hunk 2]
```diff
@@ -4,8 +4,15 @@ First, clone a copy of the main git repo by running:
...
+    # npm install -g grunt-cli
     npm install
-    grunt
 
-The built version of osgjs will be put in the `builds/` subdirectory.
\ No newline at end of file
+    grunt build
+    # The built version of osgjs will be put in the `builds/` subdirectory
+
+    # use 'webpack -w' when you are develping to rebuild automatically when a file change
```
Comment (jtorresfabra) [inline website/contents/installation.md]: s/develping/developing

**Pull request merged**